### ISSUE TITLE
Vulnerabilities in transitively referenced package System.Private.Uri (Version 4.3.0)

### ISSUE THREAD

**Issue by ManuelHaas** [groupdocs-viewer/GroupDocs.Viewer-for-.NET-UI#62]
According to NuGet package manager 3 transitive vulnerabilities in GroupDocs.Viewer.UI.SelfHost.Api version 6.0.31 exists.
That package references System.Private.Uri in version 4.3.0 that has 2 as high and one as moderate rated severities:
https://github.com/advisories/GHSA-5f2m-466j-3848
https://github.com/advisories/GHSA-xhfc-gr8f-ffwc
https://github.com/advisories/GHSA-x5qj-9vmx-7g6g

- .NET Core version 8.0.11
- GroupDocs.Viewer.UI version 6.0.31

![image](https://github.com/user-attachments/assets/da80bd05-71cb-44a1-9f89-1cca6f52b497)

**Comment by vladimir-litvinchik** [comment]
@ManuelHaas 

Thank you for reporting this. We'll take a look and update dependency versions.

**Comment by ManuelHaas** [comment]
@vladimir-litvinchik 

The release notes of the latest release states: "remove vulnerabilities in dependencies."
But the vulnerabilities are not fixed by that release.

**Comment by ManuelHaas** [comment]
According to Visual Studio 17.12.2 the transitive reference still exists:
![image](https://github.com/user-attachments/assets/31bf5484-3406-4f92-8e1c-1393a80397b1)


Note: The blacked-out libraries are those of ours that directly or indirectly reference GroupDocs.

**Comment by vladimir-litvinchik** [comment]
@ManuelHaas 

I have updated Visual Studio to version 17.12.3. Created two web applications, first one is .NET 6 and second one is .NET 8. Unfortunately I can't find any vulnerable packages in NuGet package explorer.

<img width="959" alt="Screenshot 2024-12-06 140019" src="https://github.com/user-attachments/assets/7a071eed-847f-423d-81f0-1a7ff25e55f1">

Also, I have checked generated deps.json files but haven't fond System.Private.Uri dependency.

This is the sample apps:  [web-apps.zip](https://github.com/user-attachments/files/18037910/web-apps.zip)

May you modify or create a new app to that reproduces this issue.
